src/Proto/nodeguard.proto

```diff
@@ -58,4 +63,19 @@ message RequestWithdrawalResponse {
   //  optionalTransaction ID of the withdrawal
   string txid = 1;
   bool is_hot_wallet = 2;
+}
+
+message GetAvailableWalletsRequest {}
```

Comment:
I wonder if we could generalise this method a little bit so we can add filters so we can use the same method instead of having to write future ones such as getAllHotWallets, getWalletbyId, getAllColdWallets, etc.
Food for thought

Reply:
I needed both hot and cold wallets at the same time in the btcpayserver so I just did one call and filtered in the receiving end. But we can change it if you need it for your task